src/libraries/System.Private.CoreLib/src/System/SpanHelpers.T.cs

```diff
@@ -1342,5 +1177,905 @@ public static int SequenceCompareTo<T>(ref T first, int firstLength, ref T secon
             }
             return firstLength.CompareTo(secondLength);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
You can also delete all AggressiveOptimization attributes on these methods while you are on it. They should not be needed anymore.